docs/manual/forms/variables.rst

```diff
@@ -0,0 +1,170 @@
+auth.value         string    ``111222333``               De identificerende waarde in het ``attribute`` van de
+                                                         authenticatie plugin.
+auth_type          string    ``bsn``                     Kan de waarden ``bsn``, ``kvk`` of ``pseudo`` hebben.
+auth_bsn           string    ``111222333``               De waarde van ``auth.value`` indien ``auth_type`` als waarde
+                                                         ``bsn`` heeft. Anders leeg.
+auth_kvk           string    ``90001354``                De waarde van ``auth.value`` indien ``auth_type`` als waarde
+                                                         ``kvk`` heeft. Anders leeg.
+auth_pseudo        string    ``a8bfe7a293dd``...         De waarde van ``auth.value`` indien ``auth_type`` als waarde
+                                                         ``pseudo`` heeft. Anders leeg.
+auth_context       object    ``{"source": "...", ...}``  De volledige authenticatiecontext, met machtigingsinformatie. Zie
+                                                         :ref:`manual_forms_variables_auth_context` voor de beschrijving.
+================== ========= =========================== =========================================================================
+
+**Verouderde variabelen**
+
+Deze variabelen zijn nog wel beschikbaar, maar we raden aan om deze niet meer te
+gebruiken. In versie 3.0 van Open Formulieren kunnen deze verwijderd worden.
+
+=============== ========= =========================== =========================================================================
+Variabele       Type      Voorbeeldwaarde             Toelichting
+=============== ========= =========================== =========================================================================
+auth.machtigen  object                                Verouderd, wordt in 3.0 verwijderd.
+auth.attribute  string    ``bsn``                     Kan de waarden ``bsn``, ``kvk`` of ``pseudo`` hebben (verouderd,
+                                                      gebruik bij voorkeur ``auth_type``).
+=============== ========= =========================== =========================================================================
+
+.. _manual_forms_variables_auth_context:
+
+Authenticatiecontext
+--------------------
+
+De "authenticatie context" bevat alle beschikbare informatie over *hoe* iemand ingelogd
+is op een formulier. Deze bundel informatie is beschikbaar in de vaste variabele
+``auth_context``.
+
+Wanneer er niet ingelogd is op het formulier, dan is de waarde van deze variabele
+``null``.
+
+.. note:: Het is een nieuwere en volledigere variant van de ``auth`` variabele.
+   Er zijn geen plannen om deze laatste te verwijderen.
+
+De variabele bevat een bak aan informatie, gestructureerd volgens het
+authenticatiecontextdatamodel (TODO: add link). De structuur is als volgt:
```

Comment:
will this be added later?

Reply:
yes, though I need to check if I can share those gitbook links and people outside of our team probably need to review/tweak the content, so I don't think we should block this PR on that TODO.